test/requirements.txt

```diff
@@ -13,3 +13,4 @@ protobuf >= 3.9.2, < 3.20
 datasets
 graphviz
 adlfs
+awscli>=1.27.66
```

Comment:
Question: curious why we need to specify the version here?
1. I assume it doesn't work with version prior to `1.27.66`?
2. Does it automatically try to install a version before `1.27.66`?

Reply:
Hi @NivekT, actually the version doesn't matter here.

Reply:
Hi @NivekT, actually the version doesn't matter here as far as I know. I pinned it since I noticed that pip install -r requirements.txt would download and cache all versions of awscli, which is taking a bit long. Do you recommend reverting this or haivng some other edit advice?

Reply:
> pip install -r requirements.txt would download and cache all versions of awscli

Emmm. I am not aware of that. I think it would only download the latest version.

Reply:
This is what I see when I try the command locally. It only installs the latest version though. Please let me know if I need to revert this change, as well as other changes I need to make based on the test results! @ejguan @NivekT

Reply:
@ejguan Are you fine with keeping the version number here?

Reply:
not a big deal as it's `>=` not `==`.